config/bootstrap.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require dirname(__DIR__).'/vendor/autoload.php';
+
+//if (!class_exists(Dotenv::class)) {
+//    throw new LogicException('Please run "composer require symfony/dotenv" to load the ".env" files configuring the application.');
+//}
+//
+//// Load cached env vars if the .env.local.php file exists
+//// Run "composer dump-env prod" to create it (requires symfony/flex >=1.2)
+//if (is_array($env = @include dirname(__DIR__).'/.env.local.php') && (!isset($env['APP_ENV']) || ($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? $env['APP_ENV']) === $env['APP_ENV'])) {
+//    (new Dotenv(false))->populate($env);
+//} else {
+//    // load all the .env files
+//    (new Dotenv(false))->loadEnv(dirname(__DIR__).'/.env');
+//}
```

Comment:
It's possible now to use the env files, but I didn't want to do that to prevent refactoring the configuration in Deploy. I've removed the code but added a docblock to the source of the original file.